src/main/java/com/solace/samples/TopicPublisher.java

```diff
@@ -17,6 +17,10 @@
  * under the License.
  */
 
+/**
+ *  Solace JMS 1.1 Examples: TopicPublisher
```

Comment:
Naming. This should be QPID or Something... We need to be sure we are making a differentiation between Solace JMS and ApacheQpid JMS 1.1